### ISSUE TITLE
PHP Fatal Error in ratchet/.../WsServer.php function

### ISSUE THREAD

**Issue by RynnHeldeD** [ratchetphp/Ratchet#330]
Hello !

I'm using this library for weeks, having no problem or particular issue until today. I made a test with a few friends to test our application WebSocket server with a heavier charge and it crashes when someone connected, giving this error :

```
PHP Notice:  Undefined property: stdClass::$version in /home/myProject/websockets/vendor/cboden/ratchet/src/Ratchet/WebSocket/WsServer.php on line 114
PHP Fatal error:  Call to a member function handshake() on a non-object in /home/myProject/websockets/vendor/cboden/ratchet/src/Ratchet/WebSocket/WsServer.php on line 114
```

Below is the code for the WS server instanciation :
```php
<?php
require dirname(__DIR__) . '/vendor/autoload.php';

use Ratchet\Server\IoServer;
use Ratchet\Http\HttpServer;
use Ratchet\WebSocket\WsServer;
use myProject\App;
use myProject\Model\Tool;
require ('src/App.php');

    $server = IoServer::factory(
        new HttpServer(
            new WsServer(
                new App()
            )
        ),
        8080
    );

	try {		
		$server->run();
	} catch(\Exception $e) {
		Tool::log('An error has occurred: {' . $e->getMessage() . '}', 'fatal');
	}
```

The error message prompts in the log before the log I placed in "onOpen" and "onError" methods so I supposed the error occured just before the "onOpen" phase, while creating the connection.

I don't know what exactly happened but my friends just said they tried to connect at the same time.
Any idea or fix ?

Thanks

**Comment by cboden** [comment]
Same as #238 

Are you able to consistently replicate this issue or was this a one time thing?

**Comment by RynnHeldeD** [comment]
Hi Chris

Sorry for the duplicate, I did not see this issue.

For now, it was a one time bug. We made tons of tests when developing our application with your library and I never get this error until yesterday, when we open our beta test. I'll ask my friends to try to reproduce it but I don't know exactly what they've done and they don't either.

I'll keep you inform if we managed to reproduce it consistently.

And thanks for your nice lib !
